snippets/en/_includes/self-managed-object-storage-requirements.mdx

```diff
@@ -0,0 +1,11 @@
+W&B requires object storage with pre-signed URL and CORS support.
+
+For production deployments, W&B recommends using managed object storage services:
+- [Amazon S3](https://aws.amazon.com/s3/) - Object storage service offering industry-leading scalability, data availability, security, and performance.
+- [Google Cloud Storage](https://cloud.google.com/storage) - Managed service for storing unstructured data at scale.
+- [Azure Blob Storage](https://azure.microsoft.com/products/storage/blobs) - Cloud-based object storage solution for storing massive amounts of unstructured data.
+- [CoreWeave AI Object Storage](https://docs.coreweave.com/docs/products/storage/object-storage) - High-performance, S3-compatible object storage service optimized for AI workloads.
+
+For self-hosted object storage options, see the [bare-metal guide object storage section](/platform/hosting/self-managed/bare-metal/#object-storage) for detailed setup instructions including CORS configuration, MinIO deployment, and enterprise alternatives.
```

Comment:
We may need to add a warning that Minio Open Source is no longer available.
https://github.com/minio/minio?tab=readme-ov-file#maintenance-mode